optuna/visualization/_param_importances.py

```diff
@@ -24,6 +25,64 @@
 logger = get_logger(__name__)
 
 
+class _ImportancesInfo(NamedTuple):
+    title: str
+    x_axis_name: str
+    y_axis_name: str
+    importance_values: List[float]
+    param_names: List[str]
+    importance_labels: List[str]
+
+
+def _get_importances_info(
+    study: Study,
+    evaluator: Optional[BaseImportanceEvaluator],
+    params: Optional[List[str]],
+    target: Optional[Callable[[FrozenTrial], float]],
+    target_name: str,
+) -> _ImportancesInfo:
+    _check_plot_args(study, target, target_name)
+
+    title = "Hyperparameter Importances"
+    x_axis_name = f"Importance for {target_name}"
+    y_axis_name = "Hyperparameter"
+
+    # Importances cannot be evaluated without completed trials.
+    # Return an empty figure for consistency with other visualization functions.
```

Comment:
A nit, but we're no longer dealing with figures after this function call has been moved, so this comment lies a bit. I think it can be safely removed, or rephrased.

Reply:
Nice catch. Indeed, this warning message explains this part, so let's remove this!